index.html

```diff
@@ -48,6 +48,48 @@
             <div class="bg-d"></div>
         </nav>
     </div>
+    <div class="m-window" id="m-window">
+        <div class="close-btn-container">
+            <a class="m-close-btn"><i class="fas fa-times  close-btn" id="close-btn"></i></a>
+            <h3 class="p-title" id="pd-title">Project name goes here...</h3>
+        </div>
+        <div class="w-detail-container">
+            <h3 class="p-title" id="p-title">Project name goes here...</h3>
+            <div class="l-items" id="l-items">
+                <div>Html/Css</div>
+                <div>Ruby on Rails</div>
+                <div>JavaScript</div>
+            </div>
+            <div class="b-laptop">
+                <img id="mw-img" src="images/group-105.png" alt="big laptop">
+            </div>
+            <p class="p-description">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit
+                amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi.
+            </p>
+            <div class="btn-window" id="btn-window">
+                <a class="btn-live" id="btn-live" href="#" target="_blank">See Live <img id="l-img" src="images/ic_link_teal.png"
+                    alt="see live icon"></a>
+                <br />
+                <a class="btn-source" id="btn-source" href="#" target="_blank">See Source <img id="s-img" src="images/ic_github.png"
+                    alt="see source icon"></a>
+            </div>
+            <div class="btn-window" id="d-btn-window">
+                <a class="btn-live" id="d-btn-live" rel="noopener" href="https://gabrielcoder247.github.io/" target="_blank">See Live <img id="l-img-btn" src="images/ic_link_teal.png"
+                    alt="see live icon"></a>
+                <br />
```

Comment:
- [ ] The modal popup should be populated from the array of objects containing all your projects. Delete this code from the HTML structure.